Sources/BraintreePayPalMessaging/BTPayPalMessagingPageType.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import PayPalMessages
+
+/// Message location within an application
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingPageType {
+
+    /// Home view
+    case home
+
+    /// Individual product details view
+    case productDetails
+
+    /// Shopping cart view
+    case cart
+
+    /// Popover shopping cart view that covers part of the view
+    case miniCart
+
+    /// Checkout view
+    case checkout
+
+    /// Search results
+    case searchResults
+
+    var pageTypeRawValue: PayPalMessagePageType {
```

Comment:
It would be nice if we could just expose specific classes from the PayPalMessages library to our merchants.

Reply:
Also, TBH - at this stage (no pilot merchants), I would prefer having merchants directly integrate with the PayPalMessages library versus have our SDK be a tight wrapper around it's public API.

Or even having a "module" in our SDK that merely exposes the PayPalMessages library. The decision to wrap it was mostly from product pressure on the Messaging side.

Very much an aside/not relevant to this PR - but wanted to give you context.